repository/config/ca.pan

```diff
@@ -37,8 +40,10 @@ include { 'quattor/functions/repository' };
 
 '/software/repositories' = {
   if ( is_defined(YUM_CA_RPM_SNAPSHOT_DATE) && (QUATTOR_RELEASE >= '14') ) {
+    debug(format('%s: adding CA repository for YUM snapshot %s',OBJECT,YUM_CA_RPM_SNAPSHOT_DATE));
     add_repositories(SECURITY_CA_RPM_REPOSITORY,YUM_CA_RPM_SNAPSHOT_NS);
```

Comment:
YUM_CA_RPM_SNAPSHOT_NS is always defined. We must add this specific repository only if you use a QUATTOR_RELEASE >= '14' and SNAPSHOT_DATE is defined.